open_city_profile/exceptions.py

```diff
@@ -9,9 +9,13 @@ class ServiceAlreadyExistsError(ProfileGraphQLError):
     """Service already connected for the user"""
 
 
+class TokenExpiredError(ProfileGraphQLError):
+    """Token has expired"""
+
+    
 class CannotDeleteProfileWhileServiceConnectedError(ProfileGraphQLError):
     """Profile cannot be deleted while service is still connected"""
 
 
 class ProfileDoesNotExistError(ProfileGraphQLError):
-    """Profile does not exist"""
+    """Profile does not exist"""
```

Comment:
newline missing

Reply:
Yeah. Tried to be lazy and resolve conflicts with web editor. Never again 😄